File: mo-common/mo-common-security/src/main/java/ink/ckx/mo/common/security/config/IgnoreUrlProperties.java
package ink.ckx.mo.common.security.config;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

import java.util.List;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/23
 */
@Data
@ConfigurationProperties(prefix = "security")
@Component
public class IgnoreUrlProperties {

    private List<String> whitelistPaths;
}

File: mo-common/mo-common-security/src/main/kotlin/ink/ckx/mo/common/security/config/IgnoreUrlProperties.kt
package ink.ckx.mo.common.security.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/23
 */
@ConfigurationProperties(prefix = "security")
@Component
class IgnoreUrlProperties(val whitelistPaths: List<String>)